src/app.ts

```diff
@@ -0,0 +1,14 @@
+import TYPES from './types'
+import iocContainer from './ioc'
+import { SNSMessage, SNSHandler, SNSEvent } from 'aws-lambda'
+import EventNotifierSNSMessageService from './services/eventNotifierSNSMessageService'
+
+const eventNotifierSNSMessageService: EventNotifierSNSMessageService = iocContainer.get<EventNotifierSNSMessageService>(TYPES.EventNotifierSNSMessageService)
+
+export const handler: SNSHandler = async(event: SNSEvent) => {
```

Comment:
can a return type be added here, either promise<void> or whatever handleMessage returns if it can be returned instead
```suggestion
export const handler: SNSHandler = async (event: SNSEvent): Promise<void> => {
```

see comment below in the handler - don't think this needs to be async